applications/opentelemetry-operator/0.93.0/helmrelease/opentelemetry.yaml

```
@@ -0,0 +1,37 @@
+---
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: OCIRepository
+metadata:
+  name: opentelemetry-operator
```

```suggestion
  name: opentelemetry-0.93.0-operator
```
fyi starting 2.17 we can use `${releaseName}-${appVersion}-operator` but since this is targetted for 2.16 and 2.17 we can't do that here.

---

This ensures upgrades work as expected in future.

---

This makes to update more fields for the upgraded version, from 2.17 we can take as `${releaseName}-${appVersion}-operator`